functests/0_config/config.go

```diff
@@ -31,64 +37,22 @@ var _ = Describe("[performance][config] Performance configuration", func() {
 
 	It("Should successfully deploy the performance profile", func() {
 
-		var performanceProfile *performancev1alpha1.PerformanceProfile
-		var performanceMCP *mcv1.MachineConfigPool
-
-		reserved := performancev1alpha1.CPUSet("0")
-		isolated := performancev1alpha1.CPUSet("1-3")
-		hugePagesSize := performancev1alpha1.HugePageSize("1G")
-
-		cnfRoleLabel := fmt.Sprintf("%s/%s", testutils.LabelRole, utils.RoleWorkerCNF)
-		nodeSelector := map[string]string{cnfRoleLabel: ""}
-
-		performanceProfile = &performancev1alpha1.PerformanceProfile{
-			TypeMeta: metav1.TypeMeta{
-				Kind:       "PerformanceProfile",
-				APIVersion: performancev1alpha1.SchemeGroupVersion.String(),
-			},
-			ObjectMeta: metav1.ObjectMeta{
-				Name: utils.PerformanceProfileName,
-			},
-			Spec: performancev1alpha1.PerformanceProfileSpec{
-				CPU: &performancev1alpha1.CPU{
-					Reserved: &reserved,
-					Isolated: &isolated,
-				},
-				HugePages: &performancev1alpha1.HugePages{
-					DefaultHugePagesSize: &hugePagesSize,
-					Pages: []performancev1alpha1.HugePage{
-						{
-							Size:  "1G",
-							Count: 1,
-							Node:  pointer.Int32Ptr(0),
-						},
-					},
-				},
-				NodeSelector: nodeSelector,
-				RealTimeKernel: &performancev1alpha1.RealTimeKernel{
-					Enabled: pointer.BoolPtr(true),
-				},
-				AdditionalKernelArgs: []string{
-					"nmi_watchdog=0",
-					"audit=0",
-					"mce=off",
-					"processor.max_cstate=1",
-					"idle=poll",
-					"intel_idle.max_cstate=0",
-				},
-				NUMA: &performancev1alpha1.NUMA{
-					TopologyPolicy: pointer.StringPtr("single-numa-node"),
-				},
-			},
+		performanceProfile := testProfile()
+		performanceManifest, found := os.LookupEnv("PERFORMANCE_PROFILE_MANIFEST_OVERRIDE")
```

Comment:
for a followup: all the env variables we use should be documented _somewhere_ or we will forget very fast